packages/reactive-element/README.md

```diff
@@ -105,7 +98,7 @@ $ npm i -D @webcomponents/webcomponentsjs
 
 ## Development mode
 
-@lit/reactive-element includes a development mode which adds additional checks that are
+`@lit/reactive-element` includes a development mode which adds additional checks that are
 reported in the console.
 
 To enable development mode, add the `development` exports condition to your node
```

Comment:
https://github.com/rollup/plugins/pull/540 is merged; do we need to include the NOTE here? It shows up in all the readmes.

Reply:
Removed.